requirements.txt

```diff
@@ -12,7 +12,7 @@ httpx>=0.25.0
 instructlab-eval>=0.0.3
 instructlab-quantize>=0.1.0
 instructlab-schema>=0.2.0
-instructlab-sdg>=0.0.4
+instructlab-sdg>=0.0.4.1
```

Comment:
Not blocking this PR, just a quick question: what is the reason for doing this vs. just bumping to 0.0.5?

Reply:
because we've merged a bunch of stuff to `main` that I didn't want to release yet. I made `v0.0.4.1` that is one tiny trivial change on top of `v0.0.4` to allow the `pipeline` parameter to be specified, though it doesn't actually support any other pipeline types yet (in v0.0.4.1 anyway, it does in `main`)

Reply:
https://github.com/instructlab/sdg/compare/v0.0.4...v0.0.4.1

Reply:
^^^ the release, vs what's going to be in v0.0.5 -- https://github.com/instructlab/sdg/compare/v0.0.4...main

Reply:
Ah I see. I didn't realize that you could have version numbers beyond the standard semver ones in PyPi.

Reply:
Yeah, I had to check before  I did it.

https://peps.python.org/pep-0440/

> The canonical public version identifiers MUST comply with the following scheme:
> 
> `[N!]N(.N)*[{a|b|rc}N][.postN][.devN]`

or the important piece of it:

> Release segment: `N(.N)*`

so it looks like you can have as many `.N` segments as you want

Reply:
X.Y.Z.??? :wink:

Reply:
@nathan-weinberg what’s do you mean?

anyway this release will probably last 1-2 days max

Reply:
I was just wondering what would come after Z (to be clear I have no issue with this)

Reply:
That depends on your native language. For Swedish people it's `å`, `ä`, and `ö`. In German, `z` is the last letter because `ä` comes after `a` (`ä` is sometimes transliterated to `ae`).

Reply:
That would be fun but hard to write on my US keyboard :joy: